api/model/account.go

```diff
@@ -15,7 +15,7 @@ var (
 	JwtSecret         = []byte("more secrets!")
 	Issuer            = "eqip"
 	BasicAuthAudience = "Basic"
-	Expiration        = time.Hour * 24
+	Expiration        = time.Hour * 1
```

Comment:
What does it mean that the token expires ever hour instead of every day?

Reply:
We had set it to 24 hours when there were some issues and we needed it to last for the duration of usability testing. In reality this will most likely be changed to something less than one hour in the future depending on production requirements.